app/forms/decidim/consultations/admin/question_form.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module Decidim
```

Comment:
can you document that using a decorator with class eval did not work?
also, can you document what is this override changing regarding the original¿

Reply:
I do it with a decorator! YEEAH!